tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AutoInstrumentationExtensions.cs

```diff
@@ -11,22 +12,24 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation
     internal static class AutoInstrumentationExtensions
     {
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal static void DisposeWithException(this Scope scope, Exception exception)
+        internal static void DisposeWithException(this Scope? scope, Exception? exception)
         {
-            if (scope != null)
+            if (scope == null)
```

Comment:
Why the flip? I know Rider tells you to but...

Reply:
looks better with the extra indentation?